changelogs/changelog.yaml

```diff
@@ -82,9 +82,25 @@ releases:
         namespace: null
     release_date: '2020-06-23'
   1.0.1:
+    changes: {}
+    fragments: []
+    release_date: '2020-06-23'
+  1.0.4:
     changes:
+      bugfixes:
+      - Make `src`, `backup` and `backup_options` in iosxr_config work when module
+        alias is used (https://github.com/ansible-collections/cisco.iosxr/pull/63).
+      - Makes sure that docstring and argspec are in sync and removes sanity ignores
+        (https://github.com/ansible-collections/cisco.iosxr/pull/62).
+      - Update docs after sanity fixes to modules.
       minor_changes:
-      - Bring plugin table to correct position (https://github.com/ansible-collections/cisco.iosxr/pull/44)
+      - Add changelogs for previous releases (https://github.com/ansible-collections/cisco.iosxr/pull/47)
+      release_summary:
+      - Rereleased 1.0.3 with updated changelog.
     fragments:
-    - 44-plugin-doc-fixes.yaml
-    release_date: '2020-06-23'
+    - 1.0.4.yaml
```

Comment:
You set `keep_fragments: true`, but you haven't committed the fragment with the release summary